### PR TITLE
aosc-community-wallpapers-2020: new, 1.0.0

### PR DESCRIPTION
**Pull request by neruthes**
Topic Description
-----------------

New package: `aosc-community-wallpapers-2020` with dependency.

Package(s) Affected
-------------------

- `wallcolle-make`
- `aosc-community-wallpapers-2020`

Security Update?
----------------

No

Build Order
-----------

- `wallcolle-make`
- `aosc-community-wallpapers-2020`

Architectural Progress
----------------------

    

- [x] Architecture-independent `noarch`

Secondary Architectural Progress
--------------------------------

Architectural progress for "secondary," or experimental ports does not impede on merging of this topic.

----

After the pull request is merged, all package(s) affected must be rebuilt against the `stable` Git tree and environment (only `stable` repository should be enabled in `sources.list`). This section marks the progress above.

Please, make sure the list of architectures below matches the ones above.

Post-Merge Architectural Progress
---------------------------------

- [x] Architecture-independent `noarch`

Post-Merge Secondary Architectural Progress
-------------------------------------------

### CODE REVIEW

File: extra-wallpaper/aosc-community-wallpapers-2020/autobuild/build [hunk 1]
```diff
@@ -0,0 +1,6 @@
+abinfo "Building \"aosc-community-wallpapers-2020\"..."
+
+cd "${SRCDIR}"
+node make.js community-2020 "${PKGDIR}" NORMAL
```
Comment (MingcongBai) [inline extra-wallpaper/aosc-community-wallpapers-2020/autobuild/build]: Use absolute path here...

```
node "$SRCDIR"/make.js community-2020 "${PKGDIR}" NORMAL
```

File: extra-wallpaper/aosc-community-wallpapers-2020/spec [hunk 2]
```diff
@@ -0,0 +1,4 @@
+VER=1.0.0
+GITTAG=0.0.1-pre2
```
Comment (MingcongBai) [inline extra-wallpaper/aosc-community-wallpapers-2020/spec]: You should consider a new versioning scheme for WallColle. Two versions for one such simple package is unnecessarily complicating things.

**Review by MingcongBai — CHANGES_REQUESTED** [review]

**Comment by neruthes** [comment]
Updated. Now this topic contains 2 packages:

- wallcolle-make: 0.1.0
- aosc-community-wallpapers-2020: 1.0.0

File: extra-utils/wallcolle-make/autobuild/build [hunk 3]
```diff
@@ -0,0 +1,3 @@
+abinfo "Building \"wallcolle-make\"..."
```
Comment (MingcongBai) [inline extra-utils/wallcolle-make/autobuild/build:1]: I mean, you're clearly _installing_ it...

File: extra-wallpaper/aosc-community-wallpapers-2020/spec [hunk 4]
```diff
@@ -0,0 +1,4 @@
+VER=1.0.0
+GITTAG=20210105
```
Comment (MingcongBai) [inline extra-wallpaper/aosc-community-wallpapers-2020/spec]: Merge these versions now that you separated the make utility?

**Review by MingcongBai — CHANGES_REQUESTED** [review]

**Comment by MingcongBai** [comment]
@neruthes 两处修改意见

File: extra-utils/wallcolle-make/autobuild/build (hunk 3 above)
Comment (neruthes) [reply extra-utils/wallcolle-make/autobuild/build:1]: Aha, wording...

File: extra-wallpaper/aosc-community-wallpapers-2020/spec (hunk 4 above)
Comment (neruthes) [reply extra-wallpaper/aosc-community-wallpapers-2020/spec]: Now using date as VER

**Review by MingcongBai — APPROVED** [review]
Ship it.

**Pull request merged**